# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/libwebrtc/modules/video_capture/video_capture.h
# Commit: e85b278387fc
# Full Hash: e85b278387fc5259efead947f5069c9c2dcbffad
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2023-01-11 21:56:57
# Regressor Bug: 1806605
# File Overlap Count: 1
# Description:
#   Bug 1806605 - Pass TrackingId instead of nsCString to CaptureStage. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D165824
# ==============================================================================

diff -r 0fa64a630cc7 -r e85b278387fc third_party/libwebrtc/modules/video_capture/video_capture.h
--- a/third_party/libwebrtc/modules/video_capture/video_capture.h	Wed Jan 11 11:47:45 2023 +0000
+++ b/third_party/libwebrtc/modules/video_capture/video_capture.h	Wed Jan 11 11:47:45 2023 +0000
@@ -156,7 +156,7 @@
   virtual bool GetApplyRotation() = 0;
 
   // Mozilla: TrackingId setter for use in profiler markers.
-  virtual void SetTrackingId(const char* aTrackingId) {}
+  virtual void SetTrackingId(uint32_t aTrackingIdProcId) {}
 
  protected:
   ~VideoCaptureModule() override {}
